core/core.go

```diff
@@ -146,6 +153,7 @@ func New(config Config) (*App, error) {
 		orderWatcher:   orderWatcher,
 		ethWathcher:    ethWatcher,
 		orderValidator: orderValidator,
+		jsonSchema:     jsonSchema,
```

Comment:
```suggestion
		orderJSONSchema:     orderJSONSchema,
```

Isn't this a JSON Schema specifically for orders?

Reply:
Yep, renamed.